src/components/Logo.tsx

```
@@ -0,0 +1,32 @@
+
+const Logo = ({ size = 'small', showName = true }: ILogoProps) => {
+    const getLogoSrc = () => {
+        if (showName) {
+            return size === 'small' ? logo_lively_sm : logo_lively_lg;
+        } else {
+            switch (size) {
+                case 'small':
+                    return logo_only_sm;
+                case 'medium':
+                    return logo_only_md;
+                case 'large':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)